index.html

```diff
@@ -37,24 +30,97 @@
         display: block;
         margin: 0 auto;
       }
-    </style>
-  </head>
-  <body>
-    <!--
+  </style>
+</head>
+<body>
+  <header>
+    <img src="Media_with-removebg-preview.png" alt="Illustrated graphic of two apartments">
+    <h1>Media with Amina ♡</h1>
+    <p>3 TV genres and my favourite TV shows of each</p>
+    <nav>
+      <h2>Menu</h2>
+      <ul>
```

Comment:
At the start of the header section you should have an h2 and then for this nav menu you should have an h3